packages/compass-indexes/src/modules/index-view.ts

```diff
@@ -18,6 +24,27 @@ export default function reducer(
   state = INITIAL_STATE,
   action: AnyAction
 ): IndexView {
+  // The create index button has a dropdown where you can select regular or
```

Comment:
This is another drive-by. This is functionality we had before I refactored it, then accidentally dropped during the recent refactor.